packages/app/src/utils/formatters.ts

```diff
@@ -67,9 +64,9 @@ export function formatHexDecimals(value: string, showValueAs: HexDecimals) {
   const validValue = value === "0x" ? "0" : value;
   const prefix = isHexString(validValue) ? "" : "0x";
   if (showValueAs === "Dec") {
-    return BigNumber.from(prefix + validValue).toString();
+    return BigInt(prefix + validValue).toString();
   }
-  return BigNumber.from(prefix + validValue).toHexString();
+  return `0x${BigInt(prefix + validValue).toString(16)}`;
```

Comment:
Same

Reply:
Added an util function and used it in places like this.